src/modules/NtpClient.rb

```diff
@@ -676,6 +676,9 @@ def Import(settings)
         end
         next deep_copy(p)
       end
+
+      @ntp_records.each { |record| record["address"].strip! }
```

Comment:
why not others? e.g. type cannot have whitespaces after it? and what about options?